src/Uri.php

```diff
@@ -309,7 +309,7 @@ private function filterQueryAndFragment($str): string
             throw new \InvalidArgumentException('Query and fragment must be a string');
         }
 
-        return \preg_replace_callback('/(?:[^' . self::CHAR_UNRESERVED . self::CHAR_SUB_DELIMS . '%:@\/\?]++|%(?![A-Fa-f0-9]{2}))/', [__CLASS__, 'rawurlencodeMatchZero'], $str);
+        return \preg_replace_callback('/(?:[^'.self::CHAR_UNRESERVED.self::CHAR_SUB_DELIMS.'%:@\/\?]++|%(?![A-Fa-f0-9]{2}))/', [__CLASS__, 'rawurlencodeMatchZero'], $str);
```

Comment:
And this is against formatting PSR.

Reply:
These changes are automatic. I need to fine the correct php-cs-fixer rule to change this. 

I do not want to undo your changes.